apiserver/facades/agent/uniter/uniter.go

```diff
@@ -135,18 +135,6 @@ func (u *UniterAPI) EnsureDead(ctx context.Context, args params.Entities) (param
 			continue
 		}
 
-		// TODO(units) - remove me.
-		// Dual write dead status to state.
-		unit, err := u.getLegacyUnit(ctx, tag)
-		if err != nil {
-			result.Results[i].Error = apiservererrors.ServerError(err)
-			continue
-		}
-		if err = unit.EnsureDead(); err != nil {
```

Comment:
We can't remove this without an equivalent.

Reply:
Its done below! This was a duel write.

I should probably have done this in a separate PR tbh, forgot it was in here.